src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -135,9 +160,35 @@ export default class OrganizationIntegrations extends AsyncComponent {
       );
     });
 
+    const unmigratableWarnings = map(this.unmigratableReposByOrg, (repos, org) => {
+      // VSTS Repositories used to use 'visualstudio' as the provider key. Now we use 'vsts'.
+      const providerKey =
+        repos[0].provider.id == 'visualstudio' ? 'vsts' : repos[0].provider.id;
+
+      const providers = this.state.config.providers;
+      const provider = find(providers, p => p.key == providerKey);
+
+      const integration = find(
```

Comment:
You don't have to use lodash's find here. Just use [`Array.prototype.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)